openedx_caliper_tracking/tests/local_test_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# This script file is to run the caliper app tests locally.
+
+
+FOLDER_NAME="caliper_app_testing"
+PROJECT_NAME="caliper_project"
+rm -rf $FOLDER_NAME
+mkdir $FOLDER_NAME
+cd $FOLDER_NAME
+
+printf "\n\n\n=========================================\n"
+printf "Making and Activating Virtual Environment"
+printf "\n=========================================\n"
+virtualenv venv --python=python2.7
+source venv/bin/activate
+
+printf "\n\n\n=======================\n"
+printf "Installing Dependencies"
+printf "\n=======================\n"
+pip install celery==4.3.0
+pip install Django==1.11.21
+pip install isodate==0.6.0
+pip install kafka-python==1.4.7
+pip install mock==3.0.5
+pip install python-dateutil==2.8.0
+pip install requests==2.22.0
+pip install coverage
+pip install factory-boy==2.12.0
+pip install git+https://github.com/ucsd-ets/openedx-caliper-tracking.git@umar/make-caliper-tests-independent
```

Comment:
Shouldn't we add these dependencies in a requirements file and install it through that file?

Reply:
done.